controllers/suite_test.go

```diff
@@ -8,7 +8,6 @@ import (
 	"testing"
 
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 	"k8s.io/client-go/kubernetes/scheme"
```

Comment:
Not sure why the `gomega` line was removed, but I think it's causing the CI failures.

```suggestion
        . "github.com/onsi/gomega"
	"k8s.io/client-go/kubernetes/scheme"
```